### PR TITLE
FAQ: add immutability of host files question

### LINKED ISSUE #103: Create FAQ: Security: Does Bottlerocket’s immutability mean that no files will ever change on the host?
talk about (and link) to stuff about SELinux for non-root fs and dm-verity for root fs

### PR DESCRIPTION
**Pull request by mchaker**
**Issue number:**

Closes #103 

**Description of changes:**

Adds a FAQ item that explains the range of mutability on a Bottlerocket filesystem.

**Terms of contribution:**

By submitting this pull request, I confirm that my contribution is made under
the terms of the licenses outlined in the LICENSE-SUMMARY file.

### CODE REVIEW

File: content/en/faqitems/security-1-immutability-files-change-on-host.markdown [hunk 1]
```diff
@@ -0,0 +1,14 @@
++++
+question = "Does Bottlerocket’s immutability mean that no files will ever change on the host?"
+group = "Security"
++++
+
+There are both immutable and mutable areas on a Bottlerocket host.
```
Comment (stockholmux) [inline content/en/faqitems/security-1-immutability-files-change-on-host.markdown:6]: You don't directly answer the question if you keep it as yes/no.

File: content/en/faqitems/security-1-immutability-files-change-on-host.markdown [hunk 2]
```diff
@@ -0,0 +1,14 @@
++++
+question = "Does Bottlerocket’s immutability mean that no files will ever change on the host?"
```
Comment (stockholmux) [inline content/en/faqitems/security-1-immutability-files-change-on-host.markdown:2]: Can you rework the question to better fit answer? (The answer is good.. question maybe could use work).

**Review by stockholmux — CHANGES_REQUESTED** [review]
I think the question could be reworded. If you strongly like the question, then you need to directly answer the question (yes or no and then expand).

File: content/en/faqitems/security-1-immutability-files-change-on-host.markdown (hunk 1 above)
Comment (mchaker) [reply content/en/faqitems/security-1-immutability-files-change-on-host.markdown:6]: Rephrasing the question to: "I heard Bottlerocket is immutable. What does that really mean?"

**Review by stockholmux — APPROVED** [review]
lgtm

**Pull request merged**